ooni/scripts/ooniresources.py

```diff
@@ -38,7 +30,5 @@ def run():
         print "%s: Try --help for usage details." % (sys.argv[0])
         sys.exit(1)
 
-    if options['update-inputs'] or options['update-geoip']:
-        print("WARNING: Passing command line arguments is deprecated")
-
-    yield update.download_resources()
```

Comment:
I actually don't want it to do anything since the management of the resource now is entirely delegated to the the scheduled tasks.

I am leaving the command though, since it may be part of peoples scripts and I don't want to break them. I am going to add to the output mention of the fact that it actually doesn't do anything.